src/org/usfirst/frc/team4915/stronghold/commands/ExtendLauncherCylinderCommand.java

```diff
@@ -0,0 +1,32 @@
+package org.usfirst.frc.team4915.stronghold.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+import org.usfirst.frc.team4915.stronghold.Robot;
+
+public class ExtendLauncherCylinderCommand extends Command {
+
+    public ExtendLauncherCylinderCommand() {
+        requires(Robot.intakeLauncher);
+    }
+
+    protected void initialize() {
```

Comment:
consider running `Code -> Clean Up...` on your src folder to automatically add the `@Override` annotations that are missing from these methods.